bescort.lic

```diff
@@ -394,23 +394,35 @@ class Bescort
       move 'go trail'
       find_room_maze
     when /elder/
-      echo 'Not implemented'
+	  desert_enter
+      move_direction?(priority_directions) until DRRoom.room_objs.include?('faint trail that stretches towards a sand-filled valley')
+      move 'go trail'
+	  move_direction?(priority_directions) until DRRoom.room_objs.include?('faint path leading to some nearby dunes')
+	  move 'go path'
+      find_room_maze
       exit
     when /exit/
       if XMLData.room_title.include?('Sand Valley') # Adult Area
         move_direction?(priority_directions) until DRRoom.room_objs.include?('faint trail leading to some nearby dunes')
         move 'go trail'
         desert(mode)
       end
-      if XMLData.room_title.include?('High Dunes') # Juvie Area
-        move_direction?(priority_directions) until DRRoom.room_objs.include?('faint trail that stretches towards a sand-filled valley')
+      if XMLData.room_title.include?('High Dunes') # Juvie Area or Elder
+        move_direction?(priority_directions) until DRRoom.room_objs.include?('faint trail that stretches towards a sand-filled valley')  || DRRoom.room_objs.include?('faint path that stretches towards a sand-filled valley')
+		if DRRoom.room_objs.include?('faint trail that stretches towards a sand-filled valley')
+		else
+		  move 'go path'
+		  desert(mode)
+		  exit
+		end
         move 'south'
```

Comment:
Should these three lines be in the `if` block?

Reply:
Yes, the move until above will stop you in the trail or path room
if 'faint trail...' it goes to move south - Juvies
else 'go path' then restarts the exit script - elders
the exit is because otherwise it loops the end twice 

I tested 
;bescort desert elders then exit 
;bescort desert adult then exit
;bescort desert juvenile then exit

Have also been running 3 hunting zones since I changed, running a stealth script in juvies, backtraining in adult, and mains in elders - hasn't had any issues yet.

I guess more specific logic would be a better standard, but I'm not sure of syntax
move until 'faint trail' || 'faint path'
if 'faint trail' continue
else if 'faint path' do the go path desert mode

Not sure if else if is a thing?